src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>Untitled</title>
+    
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+  </head>
+  <body>
+    <main>
+      <h1>Login</h1>
+      <div class="forms">
+        <form method="post" class="minimal">
```

Comment:
Nothing's going to stop this form from submitting. You'll want to add some javascript to prevent this.